LRQA-29204 | master

austinchiang opened the pull request:
https://issues.liferay.com/browse/LRQA-29204

austinchiang committed:
LRQA-29204 Add new paths

austinchiang committed:
LRQA-29204 Use global locator

austinchiang committed:
LRQA-29204 Remove unused locator

austinchiang committed:
LRQA-29204 Add Twitter macros

austinchiang committed:
LRQA-29204 Add basic Twitter test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master)/757/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-6.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/757/jenkins-report.html">here</a>.</b></p>

austinchiang commented:
@sharonchoi I've added refreshes to the Twitter test because the portlet needs to be refreshed after being configured for the new UI to appear. I've also added pauses to the test because we need to wait some time for the feed to appear. See Adolfo's comments in the description of https://issues.liferay.com/browse/LPS-69589.

@alee8888 I've switched out {{UsersAndOrganizations#IDENTIFICATION}} for NavTab#IDENTIFICATION. This is a more global path that could be also used in My Accounts (which this Twitter test uses): https://github.com/sharonchoi/liferay-portal/pull/1018/commits/37673b355a024805d210aa8497dd767ca5777d70

liferay-continuous-integration commented:
<h1>All tests PASSED.</h1><p>Build Time: 1 hour 16 minutes 29 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d3c665550435d61205a6ea6588e3664da77d9902">d3c665550435d61205a6ea6588e3664da77d9902</a></p><h4>Job Summary:</h4><ul><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master)/757/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/4504/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/70991/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-jboss70-mysql56-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/70992/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-mariadb100-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/70993/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23700/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-oracle121-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23937/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-postgresql94-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/69385/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat80-sybase160-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/63555/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-weblogic121-mysql56-jdk8/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68240/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly100-mariadb100-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/63609/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-hypersonic20-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/54045/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68241/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/1</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23938/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/10</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/70994/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/11</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23939/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/12</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23940/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/13</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/63557/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/14</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/63558/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/15</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/63559/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/16</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/70995/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/17</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68740/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/18</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/63560/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/19</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/69826/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/2</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/70996/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/20</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68741/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/21</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/69386/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/22</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/63561/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/23</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/54046/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/24</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/54047/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/25</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68093/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/26</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/69387/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/27</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/64297/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/28</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23941/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/29</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68090/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/3</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68243/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/30</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/63562/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/31</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68742/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/32</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/69827/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/33</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23395/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/34</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/60841/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/35</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/54048/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/36</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/49554/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/37</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23942/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/38</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/69388/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/39</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68242/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/4</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23943/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/40</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/64298/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/41</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68479/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/42</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68091/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/5</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68092/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/6</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23701/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/7</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/63556/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/4505/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/9</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68483/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mariadb100-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/4507/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mariadb100-jdk8/1</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/64301/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/63563/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-mysql56-jdk8/1</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/68484/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-oracle121-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/54052/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-oracle121-jdk8/1</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/4511/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-postgresql94-jdk8/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/69364/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat80-postgresql94-jdk8/1</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/63565/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/54056/">test-portal-acceptance-pullrequest-batch(master)/source-format-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-dist(master)/287/">test-portal-acceptance-pullrequest-dist(master)/jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/1317/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-6.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/757/jenkins-report.html">here</a>.</h5><h5>For upstream results, click <a href="http://test-1-1/job/test-portal-acceptance-upstream(master)/">here</a>.</h5>

sharonchoi commented:
No test changes needed :)

sharonchoi commented:
Pushed to BChan:
https://github.com/brianchandotcom/liferay-portal/pull/45752
